ZConfig/_compat.py

```diff
@@ -93,9 +100,9 @@ def exec_(code, globs=None, locs=None): #pragma NO COVER
 
 
 def raise_with_same_tb(exception):
-	"Raise an exception having the current traceback (if there is one)"
-	reraise(type(exception), exception, sys.exc_info()[2])
+    "Raise an exception having the current traceback (if there is one)"
```

Comment:
Docstrings look weird when they don't use tripple quotes `"""`.

(I'm surprised flake8 doesn't complain about that.)

Reply:
Indeed, they do!  I don't even remember re-indenting that.  Fixed quotes in commit 49e11c4.

I'm no Travis expert; pushing an attempt to add **flake8** usage as commit 5c87d9c.

Reply:
Is there any interest in using  @ambv 's  "black" instead? It seems to have lots of momentum. Projects like [virtualenv are using it](https://github.com/pypa/virtualenv/pull/1221) and it's common to set it up as a git commit hook so code is reformatted at commit time (though you still need a CI check).

Reply:
I've tried to dip my toes into `black` a few times, and I think it's too early.

The initial reformatting produces too large a diff to review comfortably (about half of all string literals get changed; I don't recall if it's from " to ' or the other way around).  I haven't used it enough to trust it without such a review (plus the "beta" designator in the version number).  Also, some of Black's stylistic choices take a while to get used to.

Reply:
It's from ' to ", so unlikely to be horrible on this codebase, and that can be turned off.

I'm still happier not applying formatting automatically.